src/content/guides/getting-started.md

```diff
@@ -173,7 +173,7 @@ __dist/index.html__
    </head>
    <body>
 -    <script src="./src/index.js"></script>
-+    <script src="main.js"></script>
++    <script src="dist/main.js"></script>
```

Comment:
HTML file is moved into `./dist` in previous step, this is wrong

Reply:
You are right. closing this